torch/nn/modules/dropout.py

```diff
@@ -181,6 +181,42 @@ def forward(self, input):
 
 
 class FeatureAlphaDropout(_DropoutNd):
+    r"""Randomly zero out entire channels (a channel is a 3D feature map,
+    e.g., the :math:`j`-th channel of the :math:`i`-th sample in the
+    batched input is a 3D tensor :math:`\text{input}[i, j]`).
+    Each channel will be zeroed out independently on every forward call with
```

Comment:
Could you mention that it is independent for each sample?

Reply:
Addressed in next update.